src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -36,15 +38,20 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="aspire" />
     <RepositoryReference Include="aspnetcore" />
+    <RepositoryReference Include="command-line-api" />
+    <RepositoryReference Include="deployment-tools" />
     <RepositoryReference Include="emsdk" />
     <RepositoryReference Include="fsharp" />
     <RepositoryReference Include="msbuild" />
-    <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
     <RepositoryReference Include="nuget-client" />
     <RepositoryReference Include="roslyn" />
     <RepositoryReference Include="runtime" />
     <RepositoryReference Include="sdk" />
+    <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <RepositoryReference Include="source-build-reference-packages" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <RepositoryReference Include="symreader" />
     <RepositoryReference Include="test-templates" />
     <RepositoryReference Include="vstest" />
```

Comment:
windowsdesktop should be in this list in non source-only cases.

Reply:
This is one of those transitive dependencies that I'd like to leave transitive because it's conditioned on being a windows build. So the sdk repo references windowsdesktop with that condition. I'd have to repeat that condition if I define it here as well. It's kind of nice only having to do that in one spot. Let me know if you feel otherwise.

Reply:
The sdk dependency is actually false. It doesn't actually have or use any of the WD properties. I think it's just there for coherency? installer redists the WD components so the dependency is 'truer' there.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/2c3836bbf7060f2963ff9deb690d1804dafcc7c9